domains/nx.json

```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "jadfoq",
+        "email": "jadfoq@getgoogleoff.me"
+    },
+    "record": {
+    "MX": [
```

Comment:
Invalid MX records. No TXT records provided? Also these appear to be NS records, could you clarify that these are MX addresses?

Reply:
https://github.com/is-a-dev/register/blob/9e02fa2a5590e83d25cd91268f4a6b2d4b81797e/domains/nx.json
https://manage.is-a.dev bug